docs/source/operations/projections/lccm.rst

```diff
@@ -0,0 +1,39 @@
+.. _lccm:
+
+********************************************************************************
+Lambert Conformal Conic (2SP Michigan)
+********************************************************************************
+
```

Comment:
Please add a table similar to this one from `merc.rst`:

https://github.com/OSGeo/proj.4/blob/3378c5ec96fb3f067beffc6c3acd15765d5ce5d1/docs/source/operations/projections/merc.rst#L12-L26

Reply:
OK.